source/common/upstream/upstream_impl.h

```diff
@@ -186,8 +203,9 @@ class HostImpl : public HostDescriptionImpl,
   }
   CreateConnectionData createConnection(
       Event::Dispatcher& dispatcher, const Network::ConnectionSocket::OptionsSharedPtr& options,
-      Network::TransportSocketOptionsSharedPtr transport_socket_options) const override;
+      const Network::TransportSocketOptionsSharedPtr transport_socket_options) const override;
```

Comment:
top level const parameter doesn't make any change, make it const ref or drop const.

Reply:
const dropped.